webapp/src/shared/components/Game/Game.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+
+import { CANVAS_WIDTH, CANVAS_HEIGHT } from './constants';
+import { GameBall, GamePaddle } from './types';
+import { initialBallState, initialPaddleState, isLose, isWin } from './physics';
+import useGameControls from './useGameControls';
+import useGameAnimation from './useGameAnimation';
+
+import './Game.css';
+
+export default function Game() {
```

Comment:
suggestion: What do you think about renaming this component to `PongTraining`/`GameTraining` and adding a button to navigate on the `PlayPage` because when we implement the real version we will move the game logic in the backend?

Reply:
For now I'll keep it as is since we don't have anything else. When we create the real game page we can move this easily it and rename it as well ;)